lib/DDG/Spice/Libraries.pm

```diff
@@ -0,0 +1,50 @@
+package DDG::Spice::Libraries;
+# ABSTRACT: Search for code on libraries.io
+
+use strict;
+use DDG::Spice;
+
+my @triggers_from_file = share('triggers.txt')->slurp;
+my @trigger_aliases;
+my %alias_hash = ();
+
+foreach my $line (@triggers_from_file){
+    chomp($line);
+    my @aliases = split(/,/, $line);
+    push(@trigger_aliases, @aliases);
+    $alias_hash{$aliases[0]} = \@aliases;
+}
+
+triggers start => @trigger_aliases;
+
+spice from => '(.+)/(.+)';
+spice to => 'https://libraries.io/api/search?platforms=$1&q=$2&api_key=34b827d44ed29b80497b1634c2e7e115';
```

Comment:
Let's replace this with an ENV Var. I'll put a key into the production machines 👍